<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>394</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fruits Quiz</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>350</height>
    </size>
   </property>
   <widget class="QWidget" name="widgetJudge" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>380</width>
      <height>220</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <widget class="QGraphicsView" name="graphicsViewjudge">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelJudge">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>100</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>382</width>
      <height>330</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutComboBOx">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxLanguage">
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>German</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Japanese</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chinese</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>190</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>190</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>377</width>
         <height>14</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelQuestion">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>What's is this?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutButton">
       <item>
        <widget class="QPushButton" name="pushButtonMelon">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Melon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonApple">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Apple</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStrawbery">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Strawberry</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>widgetJudge</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
